src/SUMMARY.md

```diff
@@ -198,6 +198,7 @@
   - [Wolfenstein: The New Order](./game-support/wolfenstein-new-order.md)
   - [Wolfenstein: The Old Blood](./game-support/wolfenstein-old-blood.md)
   - [Wolfenstein II: The New Colossus](./game-support/wolfenstein2-new-colossus.md)
+  - [Yakuza Kiwami 2](./game-support/yakuza_kiwami_2.md)
```

Comment:
Please replace underscores with dashes. As mentioned in Readme.

Reply:
Changes made.

Reply:
I see a change with PR checkboxes but the markdown filename remains the same (with underscores) still.

Reply:
Manually fixed by me.